website/content/docs/commands/plugin/runtime/list.mdx

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+page_title: plugin runtime list - Command
+description: The "plugin runtime list" command lists all available plugin runtimes in the plugin runtime catalog.
+---
+
+# plugin list
+
+Lists available plugin runtimes registered in the catalog. This does not list whether
+plugin runtimes are in use, but rather just their availability.
```

Comment:
```suggestion
plugin runtimes are in use, but rather just their availability. If
there are no registered plugin runtimes, an error is returned.
```

Reply:
We had the same note for `runtime info` so I thought we may want to add here as well.